README.md

```diff
@@ -0,0 +1,69 @@
+# React Auto Translate
+> Simple React components to automatically translate strings with Google Translate API
+
+## Installation
+```
+npm install -S react-auto-translate
+
+yarn add react-auto-translate
+```
+
+## Usage
+React Auto Translate uses React Context API to pass the translation handler around. Wrap your top component with the `<Translator/>` component.
+
+```jsx
+// App.js
+import {Translator, Translate} from 'react-auto-translate';
+
+return (
+  <Translator
+      cacheProvider={cacheProvider}
+      to='es'
+      googleApiKey='API_KEY'
+    >
+
+    <h1><Translate>Welcome!</Translate></h1>
+  ...
+  </Translator>
+);
+```
+
+## API
+### `<Translator/>` ###
+  - `cacheProvider`: optional handler to cache the translated strings.
```

Comment:
Is it worth noting that if a `cacheProvider` is not provided, it will ping google for every translation?